corehq/apps/hqwebapp/static/hqwebapp/js/lib/bootstrap-tab-history-custom.js

```diff
@@ -35,6 +35,12 @@ $(function () {
             // where State.data.tab won't be available
             link = $('a[data-toggle="tab"][href^="' + window.location.pathname + '"]');
             if (link.length !== 0) {
+                if (link.length > 1) {
```

Comment:
could benefit from a comment explaining this (i don't think i totally understand it)